torch/functional.py

```diff
@@ -1158,12 +1158,12 @@ def norm(input, p="fro", dim=None, keepdim=False, out=None, dtype=None):  # noqa
 
     @overload  # noqa: 749
     def norm(input, p="fro", dim=None, keepdim=False, out=None, dtype=None):  # noqa: 749
-        # type: (Tensor, Optional[number], Optional[List[int]], bool, Optional[Tensor], Optional[int]) -> Tensor
```

Comment:
Why are there any changes in this file ?

Reply:
These functions are getting pulled in and scripted due to recursive processing in some unit tests. This was part of the reason I added that `parse_defaults` flag and set it to `True` only for classes.